Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -1476,14 +1491,14 @@ private void ResetVisuals()
             }
 
             boxDisplay.Reset(active);
-            boxDisplay.UpdateFlattenAxis(flattenAxis);
+            boxDisplay.UpdateFlattenAxis(ActualFlattenAxis);
 
             bool isVisible = (active == true && wireframeOnly == false);
 
-            rotationHandles.Reset(isVisible, flattenAxis);
-            links.Reset(active, flattenAxis);
-            scaleHandles.Reset(isVisible, flattenAxis);
-            translationHandles.Reset(isVisible, flattenAxis);
+            rotationHandles.Reset(isVisible, ActualFlattenAxis);
+            links.Reset(active, ActualFlattenAxis);
+            scaleHandles.Reset(isVisible, ActualFlattenAxis);
+            translationHandles.Reset(isVisible, ActualFlattenAxis);
```

Comment:
small suggestion, if we want to cache `ActualFlattenAxis` throughout this block as well